doc/tutorials/outline.md

```diff
@@ -0,0 +1,90 @@
+# Tutorial Outline
+
+These guides already exists
+
+- [components](components.md)
+- [development](development.md)
+- [interactivity](interactivity.md)
+- [layouts](layouts.md)
+- [param](param.md)
+- [structure](structure.md)
+- [styling](styling.md)
+
+They should be refactored into the below
+
+## Basic
+
+These guides are for you that is just starting to discover Panel.
+
+You will learn to develop and deploy basic tools and apps that can be contained within a single Python file or notebook.
+
+WIDGET and `.bind` BASED APPROACH
```

Comment:
The more I think about it. The *Basic* section should be ... basic. We should compare it to how Streamlit and Gradio achieved success. Simplicity. In reality what Streamlit offers is really the opportunity to *bind* one function. And that is enough. Gradios `Interface` is the same. Define your inputs and get an output.